docs/Using Randomness/vrf-security-considerations.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "VRF Security Considerations"
+permalink: "docs/vrf-security-considerations/"
+---
+Gaining access to high quality randomness on-chain requires a solution like Chainlink's VRF, but it also requires you to understand some of the ways that randonmess generation can be manipulated by miners/validators. Here are some of the top security considerations you should review in your project.
+
+* [Do not depend on the order of VRF fulfillments](#do-not-depend-on-the-order-of-vrf-fulfillments)
```

Comment:
This title sounds disingenuous, I feel like it should just say "Use `requestId` to match randomness requests with their fulfillment in order"

Reply:
I think @PatrickAlphaC is making a good point, and also prefer the title he has recommended.